### PR TITLE
Improve log and allow duplicate notifications section

### PR DESCRIPTION
**Pull request by nickvergessen**
* Notifications is receiving a settings section and it's confusing for users if we add 2 different tabs for notifications and activity: https://github.com/nextcloud/notifications/pull/1074
So we want to have notifications section from both apps
* There is a log with an empty message

### CODE REVIEW

File: lib/private/Settings/Manager.php [hunk 1]
```diff
@@ -131,7 +131,7 @@ protected function getSections(string $type): array {
 
 			$sectionID = $section->getID();
 
-			if ($sectionID !== 'connected-accounts' && isset($this->sections[$type][$sectionID])) {
+			if ($sectionID !== 'connected-accounts' && $sectionID !== 'notifications' && isset($this->sections[$type][$sectionID])) {
 				$this->log->info('', ['exception' => new \InvalidArgumentException('Section with the same ID already registered: ' . $sectionID . ', class: ' . $class)]);
```
Comment (artonge) [inline lib/private/Settings/Manager.php:135]: This log also has an empty message, should it be fixed too?

**Review by artonge — CHANGES_REQUESTED** [review]

File: lib/private/Settings/Manager.php (hunk 1 above)
Comment (nickvergessen) [reply lib/private/Settings/Manager.php:135]: fixed

**Review by artonge — APPROVED** [review]
Front CI is somehow unhappy. Weird

File: lib/private/Settings/Manager.php [hunk 2]
```diff
@@ -131,8 +131,9 @@ protected function getSections(string $type): array {
 
 			$sectionID = $section->getID();
 
-			if ($sectionID !== 'connected-accounts' && isset($this->sections[$type][$sectionID])) {
-				$this->log->info('', ['exception' => new \InvalidArgumentException('Section with the same ID already registered: ' . $sectionID . ', class: ' . $class)]);
+			if ($sectionID !== 'connected-accounts' && $sectionID !== 'notifications' && isset($this->sections[$type][$sectionID])) {
```
Comment (vitormattos) [inline lib/private/Settings/Manager.php:134]: I believe will make more readable the code if put the two first conditions on a method to  check for all sessionId names

Comment (nickvergessen) [reply lib/private/Settings/Manager.php]: DOne

**Review by vitormattos — APPROVED** [review]

**Pull request merged**